test/unit_tests/modules/util/js-loader-spec.js

```diff
@@ -16,7 +16,7 @@ describe( 'loadScript method', () => {
   } );
 
   it( 'should invoke the callback method when the script loads', () => {
-    const loaderPromise = new Promise( function( resolve, reject ) {
+    const loaderPromise = new Promise( function( resolve ) {
```

Comment:
Can be arrow function. We _may_ want to keep the `reject` just so the signature of Promises is consistent, but I don't have a strong opinion on it. A similar scenario is event handlers and an unused event object in the parameters. Some might say give them all event objects so you know that's always coming into the function, others may say remove them when not needed. :man_shrugging:

Reply:
Did it anyway, I think the bit about knowing what's coming into the function makes sense.